docker/all.Dockerfile

```diff
@@ -4,51 +4,30 @@ LABEL build_date="2020-09-30" \
 	  description="Jolie Documentation" \
 	  url="https://jolie-lang.org"
 MAINTAINER Brian Alberg "alberg@imada.sdu.dk"
-## Fix issue in Node/Docker as reported here:
-## https://stackoverflow.com/questions/52196518/could-not-get-uid-gid-when-building-node-docker
-ENV npm_config_unsafe_perm true
```

Comment:
Looks like the bug in Node/Docker has been fixed, so I think it's safe to remove these lines.